n2y/plugins/downloadfileproperty.py

```diff
@@ -0,0 +1,28 @@
+import hashlib
+from os import makedirs, path
+from urllib.parse import urlparse
+
+
+from n2y.property_values import FilesPropertyValue
+from n2y.utils import slugify
+
+
+class DownloadFilePropertyValue(FilesPropertyValue):
```

Comment:
I think we need a test for this plugin. You'll probably need to extend our plugins end-to-end test ...